cyberdrop_dl/managers/client_manager.py

```diff
@@ -129,9 +132,6 @@ async def check_http_status(
             message = DOWNLOAD_ERROR_ETAGS.get(headers.get("ETag"))
             raise DownloadError(HTTPStatus.NOT_FOUND, message=message, origin=origin)
 
-        if HTTPStatus.OK <= status < HTTPStatus.BAD_REQUEST:
-            return
-
         if any(domain in response.url.host for domain in ("gofile", "imgur")):
```

Comment:
Moving this check down may break something but i'am not 100% sure. Will merge as is